app/src/components/featuredItem/campaignHero.tsx

```diff
@@ -15,9 +15,10 @@ const CampaignHero = ({ featuredItemData }) => {
   );
 
   const handleError = (e: React.SyntheticEvent<HTMLImageElement, Event>) => {
-    console.log("error loading campaign hero:", e);
+    if ((window as any).newrelic) {
+      (window as any).newrelic.noticeError("Error loading campaign hero");
```

Comment:
Can you add to the end,  " - loading default image", or something similar to that effect.